WooCommerce/Classes/ViewModels/CardPresentPayments/PaymentCaptureOrchestrator.swift

```diff
@@ -272,9 +264,7 @@ private extension PaymentCaptureOrchestrator {
 
 private extension PaymentCaptureOrchestrator {
     enum Constants {
-        static let canadaFlatFee = NSDecimalNumber(string: "0.25")
-
-        static let canadaPercentageFee = NSDecimalNumber(string: "0.026")
```

Comment:
nit: I wonder if we can keep the previous code and just change `canadaFlatFee` to `0.15` and `canadaPercentageFee` to `0.0` in case we might need the percentage in the future, perhaps for another payment method?

Reply:
That sounds right! I brought it back 😃